fix-point/src/Refactor.hs

```diff
@@ -0,0 +1,39 @@
+module Rename where 
+
+import qualified Data.Text as Universum
+import qualified Data.Text.IO as Universum
```

Comment:
Okay, usually those imports are qualified like `Data.Text as T` and `Data.Text.IO as TIO`. You shouldn't use `Universum` name here. I can see how `hlint` could you suggest that, but let's not do this.